src/sentry/snuba/referrer.py

```diff
@@ -317,7 +545,7 @@ class Referrer(Enum):
 def validate_referrer(referrer: Optional[str]):
     if not referrer:
         return
-
     referrers = {referrer.value for referrer in Referrer}
     if referrer not in referrers:
         metrics.incr("snql.sdk.api.new_referrers", tags={"referrer": referrer})
+        logger.warning(f"referrer {referrer} is not part of Referrer Enum")
```

Comment:
If you want this to create an issue in Sentry, I believe you will need to raise an Exception, catch it, and add `exc_info=` to the `logger.warning` call.